src/LuaXML_101012/Makefile

```diff
@@ -1,17 +1,21 @@
 # 2009-03-16 by gf
 #
 ifeq ($(PREFIX),)
-    PREFIX := /usr
+    PREFIX ?= /usr
 endif
 
+# Path for installation Lua modules
+# Path is relative to $(DESTDIR)
+LUAPATH ?= $(PREFIX)/lib/lua
+
 # generic compiler and linker settings:
 #CROSS_COMPILE=arm-openwrt-linux-uclibcgnueabi-
 #CC=$(CROSS_COMPILE)gcc
 
 INCDIR ?= -I../lua/src -I $(STAGING_DIR)/usr/include
 LIB    ?= 
 LIBDIR ?=  -L. -L../lua/src -L $(STAGING_DIR)/usr/lib
-override CFLAGS ?= -Os -Wall -c #-g
```

Comment:
@MuteSpirit Ivan, just a little reminder you have written me:
>But override is still needed for statements like override CFAGS += ...

Reply:
fixed